config/routes.rb

```diff
@@ -1,10 +1,12 @@
 Rails.application.routes.draw do
   root "static_pages#home"
 
-  get "/signup", to: "users#new"
-  get "/help", to: "static_pages#help"
-  get "/about", to: "static_pages#about"
-  get "/contact", to: "static_pages#contact"
+  get "users/new", to: "users#new"
+  get  "/signup", to: "users#new"
```

Comment:
```suggestion
  get "/signup", to: "users#new"
```
sau get dùng 1 spaces thôi